### PR TITLE
Update readme instructions for clarity and added details from fresh install

### PR DESCRIPTION
**Pull request by srobbibaro**
I wanted to update the instructions from my last setup/install while it was still fresh in my mind. Most of the changes are organizational, for clarify, or to expand and give more detail.

### CODE REVIEW

**Comment by srobbibaro** [comment]
@joelmccracken - quick look when you can?

File: README.md [hunk 1]
```diff
@@ -1,39 +1,66 @@
...
+
+The file should contain the following lines:
 
 ```
 export GH_CLIENT_ID=<YOUR GITHUB CLIENT ID>
 export GH_CLIENT_SECRET=<YOUR GITHUB CLIENT SECRET>
 ```
 
+Note: You may need to setup these keys through GitHub.
+For details, see: https://github.com/blog/1509-personal-api-tokens
```
Comment (joelmccracken) [inline README.md]: This isn't actually personal api tokens, these are created from here: `https://github.com/settings/applications/new`

**Comment by joelmccracken** [comment]
comment, but otherwise thanks for doing this!!

**Comment by srobbibaro** [comment]
@joelmccracken - Good catch. Updated the link to be the proper one.

**Comment by joelmccracken** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,39 +1,67 @@
...
-  * `$ git push heroku master`
+### Setup
+
+  1. Install Heroku: `$ brew install heroku`
+  2. `$ heroku login`
+  3. `$ heroku git:remote -a dev-wizard`
+
+### Deploying
+
+  To deploy to production, run: `$ git push heroku master`
```
Comment (seejee) [inline README.md:64]: Pushes to master are also deployed to Heroku by default now. It's probably worth a mention.

**Pull request merged**